### ISSUE TITLE
I2C Bus stops responding after several minutes?

### ISSUE THREAD

**Issue by Humancell** [ZIOCC/Zio-Qwiic-Loudness-Sensor-I2C#3]
I have been running your demo application, and after several minutes (3-5) of operation, the sound level board stops responding.

If I power cycle the board, it will return and continue responding for a while ... and then it locks up and stops responding.

This is occurring with several of these boards that I bought from you?

Any ideas?

**Comment by AlexChuSmartPrototyping** [comment]
Hi, there, can you share me more details about this issue so I can repeat this issue on my side? Do you mean the other loudness sensor boards also had this issue?

**Comment by 24601** [comment]
Yeah, same here....especially when the display is connected, it locks up hard after just a few seconds.

**Comment by AlexChuSmartPrototyping** [comment]
Hi I tested the board again with a display module, it works well. 
Which board are you using and how is the connection looks like? Could you share your code and a picture here or send to us an email? So we can repeat the issue and solve it. 
Thanks.

> 
> 
> Yeah, same here....especially when the display is connected, it locks up hard after just a few seconds.

**Comment by AlexChuSmartPrototyping** [comment]
I could repeat this issue on my side only by connecting the module to 5V power and I2C, the module did locked up. Could you try with 3.3V power and I2C? The Qwiic should always run on 3.3V.

**Comment by AlexChuSmartPrototyping** [comment]
We did some additional testing and comparison between 3.3V and 5V on the power and I2C bus and these are our results:

Power             Voltage | I2C Voltage | Data Refresh Rate | Result
-- | -- | -- | --
3.3V | 3.3V | Slow (~3/s) | OK
3.3V | 3.3V | Fast (~30/s) | OK
3.3V | 5.0V | Slow (~3/s) | OK
3.3V | 5.0V | Fast (~30/s) | Sometimes not working
5.0V | 5.0V | Slow (~3/s) | not OK
5.0V | 5.0V | Fast (~30/s) | not OK

Can you maybe double-check if you are getting similar results? If yes then we know it is related to the power voltage and can add that to the product description (or consider to avoid that hardware wise).
